oop/src/components/Captcha.vue

```diff
@@ -232,7 +232,11 @@ export default {
             this.audio = audio;
 
             setTimeout(() => {
-              this.$refs.audio.play();
+              //The following code makes audio playback testable in Jest.
+              //If there are problems, the old version of the code is included below
+              var snd = new Audio(this.audio);
+              snd.play();
+              // this.$refs.audio.play();
```

Comment:
A couple comments here:
1. In modern day ES6 we should use `const` where possible, and `let` for most other cases. Only place where `var` should be used is where scoping needs to be different.
2. Since local variables get minified when the build is run, I think it's best to spell out the variable by name, versus writing them shorthand. So `sound` instead of `snd` please.
3. You can remove the old `this.$refs.audio.play();` since it's no longer used.
4. Lastly a question: does using the `Audio` prototype/class work in IE11?

Reply:
1, 2, 3: Fixed in 17fb454
4: Yes, I just double checked.